src/style/landing.scss

```diff
@@ -82,14 +82,26 @@ body {
   font-size: 1.4rem;
   font-weight: 400;
   text-align: center;
-  padding-bottom: 0.8rem;
+  padding-bottom: 1rem;
 
   @include media-query($sm-up) {
     font-size: 1.8rem;
     text-align: left;
   }
 }
 
+.hero__email {
+  font-size: 1.4rem;
+  font-weight: 400;
+  text-align: center;
+  margin-bottom: 1rem;
+
+  @include media-query($sm-up) {
+    font-size: 1.4rem;
```

Comment:
Font size within and without media query are the same? Not sure what your intention was here.

Reply:
I didn't know how you set it up so I left it like that. Is that fine?